### PR TITLE
changes on the description for big data

### PR DESCRIPTION
**Pull request by kuhnen**

### CODE REVIEW

**Review by ghost — COMMENTED** [review]
typo line 4 includng
otherwise good

File: Software Engineer Big Data.md [hunk 1]
```diff
@@ -1,23 +1,14 @@
...
-# (Senior) Software Engineer – Big Data
-
-## Your Tasks
-- Design, model, implement and maintain Hadoop solutions supporting raw-event collections
 - Help internal clients with pragmatic strategic & ad-hoc data requirements
-- Optimize our data infrastructure to become a reliable, tested real-time solution
 - Work closely with data scientists and engineers to build personalized experiences for our users
 - Drive research initiatives to innovate and support our data driven approach
-- Perform data extraction, aggregation from multiple sources, includng Google Analytics
+- Perform data extraction, aggregation from multiple sources, includng Google Analytics / GoogleBigQuery
```
Comment (flaviotruzzi) [inline Software Engineer Big Data.md]: Typo: `includng` should be `including`

File: Software Engineer Big Data.md [hunk 2]
```diff
@@ -1,23 +1,14 @@
-[HolidayCheck](http://www.holidaycheck.de/) is deeply committed to providing the best experience in online travel for our customers.
-Help our growing team of data scientists and engineers to unleash the potential of billions of events every day to provide 25 million travellers every month with intelligent optimizations as a ...
-
-# (Senior) Software Engineer – Big Data
```
Comment (flaviotruzzi) [inline Software Engineer Big Data.md]: Is there any reason to delete the name of the role, and the "Your Tasks"

File: Software Engineer Big Data.md [hunk 3]
```diff
@@ -1,23 +1,14 @@
...
-- Several years of real-world experience with Apache Hadoop (with e.g. HDFS, Hive, HBase, Pig)
-- Experienced software engineer, proficient with several languages (we use Scala a lot)
-- Good command of several scripting languages, like Python, Bash, JavaScript or even R
-- Proficient with SQL (MySQL, MS-SQL are running) and NoSQL (e.g. MongoDB, Neo4j, Cassandra) databases and querying them
-- Sysadmin experience (Linux, AWS, Docker) is a plus
+- You are eager to learn new technologies
+- You are able to transform solutions into code (Scala is a plus)
+- SQL Basic skills
+- Devops mindset is a plus
 - Prior experience with scaling ETL projects and a big data platform is advantageous
```
Comment (flaviotruzzi) [inline Software Engineer Big Data.md]: I would add something like:

```
- Professional experience with at least one of the following languages: Scala, Java, C, C++ or Python.
```

File: Software Engineer Big Data.md [hunk 4]
```diff
@@ -1,23 +1,14 @@
...
 
 ## Your Profile
-- Several years of real-world experience with Apache Hadoop (with e.g. HDFS, Hive, HBase, Pig)
-- Experienced software engineer, proficient with several languages (we use Scala a lot)
-- Good command of several scripting languages, like Python, Bash, JavaScript or even R
-- Proficient with SQL (MySQL, MS-SQL are running) and NoSQL (e.g. MongoDB, Neo4j, Cassandra) databases and querying them
-- Sysadmin experience (Linux, AWS, Docker) is a plus
+- You are eager to learn new technologies
+- You are able to transform solutions into code (Scala is a plus)
+- SQL Basic skills
```
Comment (flaviotruzzi) [inline Software Engineer Big Data.md:17]: Would add
```
- Professional experience with at least one of the following DBs: PostgreSQL, Mysql, MS-SQL.
```

File: Software Engineer Big Data.md [hunk 5]
```diff
@@ -1,23 +1,14 @@
...
 - Create confidence in our data by tests, automation, monitoring and neat visualizations
 
 ## Your Profile
-- Several years of real-world experience with Apache Hadoop (with e.g. HDFS, Hive, HBase, Pig)
-- Experienced software engineer, proficient with several languages (we use Scala a lot)
-- Good command of several scripting languages, like Python, Bash, JavaScript or even R
-- Proficient with SQL (MySQL, MS-SQL are running) and NoSQL (e.g. MongoDB, Neo4j, Cassandra) databases and querying them
-- Sysadmin experience (Linux, AWS, Docker) is a plus
+- You are eager to learn new technologies
+- You are able to transform solutions into code (Scala is a plus)
```
Comment (flaviotruzzi) [inline Software Engineer Big Data.md:16]: Personally prefer: transform ideas into code.

The code would be the solution itself.

File: Software Engineer Big Data.md [hunk 6]
```diff
@@ -1,23 +1,14 @@
...
+- SQL Basic skills
+- Devops mindset is a plus
```
Comment (flaviotruzzi) [inline Software Engineer Big Data.md:18]: Would add:
```
- Clear understanding of Map/Reduce paradigm. Experience with Spark/Hadoop is a plus.
```

File: Software Engineer Big Data.md [hunk 7]
```diff
@@ -27,6 +18,8 @@ Help our growing team of data scientists and engineers to unleash the potential
 - Non-religious about specific systems, frameworks, languages or tools
 - Can focus on deliverables and small increments but also not afraid of huge changes and humongous amounts of data
 - Proud to show your github.com account, research papers or other interesting achievements
+- Functional programming skills is a plus
```
Comment (flaviotruzzi) [inline Software Engineer Big Data.md]: I do not consider functional programming or spark a personal trait, that should be described in "Your Profile" even if it is a plus.

**Comment by HeeL** [comment]
Can this be merged?

**Comment by kuhnen** [comment]
yes it can 👍

**Pull request merged**